sbc_compassion/models/contracts.py

```diff
@@ -39,6 +39,10 @@ class Contracts(models.Model):
         'Days since sponsor wrote',
         compute='_compute_last_letter'
     )
+    incite_to_write = fields.Boolean(
```

Comment:
Here is my suggestion for the field name.
```suggestion
    write_for_birthday_alert = fields.Boolean(
```